extensions/trenit/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "trenit",
+  "title": "Trenit",
+  "description": "Italian Rail departure information",
+  "icon": "extension-icon.png",
+  "author": "gianpiero_spinelli",
+  "categories": [
+    "Other"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "departures-and-arrivals",
+      "title": "Check Trains Status",
+      "description": "Lets you browse the trains status throughout Italian stations.",
+      "mode": "view",
+      "handle": "stations"
```

Comment:
Never seen the before, what does it do 😊

Reply:
oopsie, thats something i was testing... removed it, thanks!